### PR TITLE
Doc 2017 updated version numbers in ledger graph and archive service docs v4.7.1 and 4.8

### PR DESCRIPTION
**Pull request by benpesterr3**

### CODE REVIEW

**Review by erinking-r3 — DISMISSED** [review]
looks good to me

File: content/en/docs/corda-enterprise/4.7/node/archiving/archiving-setup.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ The Archive Service consists of the following:
 
 It also makes use of the [Application Entity Manager](app-entity-manager) - which allows CorDapps to access off-ledger databases using JPA APIs.
 
+## New in V1.0.1
+
+The Archive Service is compatible with [Ledger Graph V1.2.1 On Demand function](#Archiving-and-onDemand-LedgerGraph).
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.7/node/archiving/archiving-setup.md:34]: Is it only compatible with LG 1.2.1 or also with 1.2?

Comment (benpesterr3) [reply content/en/docs/corda-enterprise/4.7/node/archiving/archiving-setup.md:34]: It is compatible with the On Demand function which is only available in LG V1.2.1

File: content/en/docs/corda-enterprise/4.7/node/operating/ledger-graph.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ If this list is empty, the node has been successfully drained. If the list conta
 
 ### Uninstall Old Versions
 
-There are no earlier versions of this CorDapp. However, if you are upgrading from version 1.0 of **Collaborative Recovery**, then you should replace `ledger-sync-confidential-identities-1.0.jar` with `ledger-graph-1.1.jar`.
+There are no earlier versions of this CorDapp. However, if you are upgrading from version 1.0 of **Collaborative Recovery**, then you should replace `ledger-sync-confidential-identities-1.0.jar` with `ledger-graph-1.2.1.jar`.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.7/node/operating/ledger-graph.md:92]: The first sentence is probably slightly incorrect as a statement as it was referring to LG 1.0?

File: content/en/docs/corda-enterprise/4.7/node/archiving/archiving-setup.md (hunk 1 above)
Comment (benpesterr3) [reply content/en/docs/corda-enterprise/4.7/node/archiving/archiving-setup.md:34]: there is a link there that explains it I think

File: content/en/docs/corda-enterprise/4.7/node/operating/ledger-graph.md (hunk 2 above)
Comment (benpesterr3) [reply content/en/docs/corda-enterprise/4.7/node/operating/ledger-graph.md:92]: No - it's referring to Collaborative recovery.

Comment (benpesterr3) [reply content/en/docs/corda-enterprise/4.7/node/operating/ledger-graph.md:92]: (ledger-sync is not ledger-graph)

Comment (benpesterr3) [reply content/en/docs/corda-enterprise/4.7/node/operating/ledger-graph.md:92]: Oh - HA - I just worked out what you mean. OK

Comment (benpesterr3) [inline content/en/docs/corda-enterprise/4.7/node/operating/ledger-graph.md:92]:
```suggestion
If you are upgrading from version 1.0 of **Collaborative Recovery**, then you should replace `ledger-sync-confidential-identities-1.0.jar` with `ledger-graph-1.2.1.jar`.
```

File: content/en/docs/corda-enterprise/4.8/node/operating/ledger-graph.md [hunk 3]
```diff
@@ -89,7 +89,7 @@ If this list is empty, the node has been successfully drained. If the list conta
 
 ### Uninstall Old Versions
 
-There are no earlier versions of this CorDapp. However, if you are upgrading from version 1.0 of **Collaborative Recovery**, then you should replace `ledger-sync-confidential-identities-1.0.jar` with `ledger-graph-1.1.jar`.
+There are no earlier versions of this CorDapp. However, if you are upgrading from version 1.0 of **Collaborative Recovery**, then you should replace `ledger-sync-confidential-identities-1.0.jar` with `ledger-graph-1.2.1.jar`.
```
Comment (benpesterr3) [inline content/en/docs/corda-enterprise/4.8/node/operating/ledger-graph.md:92]:
```suggestion
If you are upgrading from version 1.0 of **Collaborative Recovery**, then you should replace `ledger-sync-confidential-identities-1.0.jar` with `ledger-graph-1.2.1.jar`.
```

**Review by ivanterziev-r3 — APPROVED** [review]
LGTM

**Pull request merged**